templates/admin.php

```diff
@@ -55,6 +55,7 @@
 			<th>Tag</th>
 			<th>Retention</th>
 			<th>Time</th>
+			<th>Active</th>
```

Comment:
Wait they are all not translated Oo

Reply:
yep noticed as well but it was like that before :shrug:

Reply:
I think that should go into another pr.